### ISSUE TITLE
Error: Failed to pull singularity image

### ISSUE THREAD

**Issue by Mechah** [bhattlab/lathe#15]
Hi, when I want to start lathe I get the following error message:

Building DAG of jobs...
Pulling singularity image shub://elimoss/lathe:longread.
WorkflowError:
Failed to pull singularity image from shub://elimoss/lathe:longread:
/usr/bin/env: ‘python’: No such file or directory

Thanks for any support!

**Comment by bsiranosian** [comment]
This is not something I've seen before... I can confirm that I can still pull the image and it works. I assume you have python and all the snakemake dependencies installed?

**Comment by Mechah** [comment]
Yep, I assume python and all snakemake dependencies are there. I'll keep on trouble shooting. If I can solve the issue, I'll let you know. Thanks for now!

**Comment by bsiranosian** [comment]
You also may want to try pulling and running a similarity image just from the command line with 'singularity she'll XXXX' where you can fill it in from the image name in the snakefie
